scripts/binary/binary-integrity-check-source.js

```diff
@@ -0,0 +1,168 @@
+const OrigError = Error
+const captureStackTrace = Error.captureStackTrace
+const toString = Function.prototype.toString
+const callFn = Function.call
+
+// eslint-disable-next-line no-unused-vars
+const stackIntegrityCheck = function stackIntegrityCheck (options) {
+  const originalStackTraceLimit = OrigError.stackTraceLimit
+  const originalStackTrace = OrigError.prepareStackTrace
```

Comment:
Do we need to create each time or can we make these global?

Reply:
This function is really only called once. I like the idea of storing this off just in time since we are replacing them immediately after. The other global concepts (Error, captureStackTrace, etc.) are ones that we don't do a replacement dance with.